src/client/components/Header/Header.tsx

```diff
@@ -1,14 +1,144 @@
 import React, { Component } from 'react';
+import './Header.css';
 
-export class Header extends Component {
-    constructor(props: any) {
-      super(props);
-      this.state = {};
+import {fetchProfileAPI} from '../../utils/API';
```

Comment:
We should run `npm run fix` (this runs prettier) before committing so formatting is consistent.

Reply:
oh cool.  i didn't know you could do that. is it something you had to set it up?